app/Libraries/BBCodeForDB.php

```diff
@@ -155,6 +155,17 @@ public function parseInlineSimple($text)
         return $text;
     }
 
+    public function parseHeading($text)
```

Comment:
https://github.com/ppy/osu-web/pull/2313/files#diff-8f78753fe51dad48708f01cc0c0723efL147 
these changes seem unnecessary.

Reply:
Oops, I forgot to change the heading function to be single-line. That's why I made it - because headings aren't supposed to be spanned across multiple lines. Fixing.

As for the multi-line modifier, I wanted to do that but I don't see where I should put it, I don't see where are the forward slashes at. Normally I have like `/something/modifiers` kind of patterns in JS, but here it's `#something#` and it throws me off. Care to hint me?

Reply:
flags goes in the same place, after the pattern delimiter:
```
/pattern/s
#pattern#s
~pattern~s
```
are equivalent.

`/` is not being used here because of `[/tag`, otherwise the `/` needs to be escaped.

Reply:
Okay, I guess it's all fine now?